### DISCUSSION TITLE
2024/04/14/install_OneNoteClipper_on_firefox/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#20] [General]
# 2024/04/14/install_OneNoteClipper_on_firefox/

（这周更新比较水，感觉最近题材有点匮乏） 2023年8月左右，微软对OneNote Web Clipper插件进行了更新——然而并不是什么好更新，而是弹出一个“OneNote Web Clipper is no longer supported on Firefox browser”的提示然后自动卸载的更新，逼着用户换Edge浏览器。 这件事实在令人烦恼。忍受了大半年无法使用这个插件进行网页一键剪

https://wz.anoms.top/2024/04/14/install_OneNoteClipper_on_firefox/

**Comment by cyclinbox** [comment]
最近发现 Poe 这个网站很好用，它是Quora（美版知乎）母公司开发的一个大模型使用平台，注册用户可以免费使用包括chatGPT、Claude3、Gemini-Pro等在内的许多AI模型，如果充会员的话还能解锁GPT4等更高级的模型。

另外我觉得最棒的功能其实是这个：创建私人bot的功能。可以选择一个基础模型（例如chatGPT），然后通过定制prompt和上传一些资料库，从而调教出一个更有特色的属于自己的机器人。（听起来和openai搞的store很像，但关键是Poe的这个功能免费）

![private bot](https://pic.imgdb.cn/item/661c11ea68eb9357137c11a9.png)

对了，还有一点就是，国内的AI里面我发现文心一言和通义千问是真的强，至少比国内其他AI要厉害好几个档次。文心一言像是个文科生，在通用知识领域很强；通义千问的数学能力和代码编写能力则强大太多。
